unity-renderer/Assets/DCLServices/PlacesAPIService/PlacesAPIClient.cs

```diff
@@ -116,6 +118,28 @@ public async UniTask<IHotScenesController.PlaceInfo> GetPlace(string placeUUID,
             return response.data;
         }
 
+        public async UniTask<List<IHotScenesController.PlaceInfo>> GetPlacesByCoordsList(List<Vector2Int> coordsList, CancellationToken ct)
+        {
+            if (coordsList.Count == 0)
+                return new List<IHotScenesController.PlaceInfo>();
+
+            var url = string.Concat(BASE_URL, "?");
+            foreach (Vector2Int coords in coordsList)
+                url = string.Concat(url, $"positions={coords.x},{coords.y}&with_realms_detail=true&");
+
+            var result = await webRequestController.GetAsync(url, cancellationToken: ct, isSigned: true);
+
+            if (result.result != UnityWebRequest.Result.Success)
+                throw new Exception($"Error fetching places info:\n{result.error}");
+
+            var response = Utils.SafeFromJson<IHotScenesController.PlacesAPIResponse>(result.downloadHandler.text);
```

Comment:
I totally agree with you: the classes `PlaceInfo`, `PlacesAPIGetParcelResponse` and `PlacesAPIService` should be moved from `IHotScenesController` to the `PlacesAPIService` folder directly, however I think this is not the best moment to do it due to the huge amount of files affected (about 33) that, some of them, are being modified by different PRs (from me and from Davide). It would be too risky to do it now because we could provoke a lot of conflicts between 2 on-going shape-up initiatives.

If you don't mind, lets move this request to a separate tech-debt task later, when we both merge our PRs 👍